src/test/java/io/vertx/cassandra/CassandraServiceBase.java

```diff
@@ -15,40 +15,100 @@
  */
 package io.vertx.cassandra;
 
+import com.datastax.driver.core.BatchStatement;
+import com.datastax.driver.core.PreparedStatement;
+import io.vertx.core.CompositeFuture;
 import io.vertx.core.Future;
 import io.vertx.core.Vertx;
 import org.apache.thrift.transport.TTransportException;
 import org.cassandraunit.utils.EmbeddedCassandraServerHelper;
 import org.junit.After;
 import org.junit.Assert;
 import org.junit.Before;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 import java.io.IOException;
 import java.util.concurrent.CountDownLatch;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+import static io.vertx.test.core.TestUtils.randomAlphaString;
 
 /**
  * A base class, which should be used for in all test where Cassandra service is required.
  */
 public class CassandraServiceBase {
 
+  private static final Logger log = LoggerFactory.getLogger(CassandraServiceBase.class);
+
   Vertx vertx = Vertx.vertx();
+
   static final String HOST = "localhost";
   static final int NATIVE_TRANSPORT_PORT = 9142;
+  private static final int BATCH_INSERT_SIZE = 1_000;
+  private static final int TIMES_TO_INSERT_BATCH = 100;
 
   @Before
-  public void before() throws InterruptedException, IOException, TTransportException {
+  public void before() throws IOException, TTransportException, InterruptedException {
     EmbeddedCassandraServerHelper.startEmbeddedCassandra();
 
-    CassandraClient cassandraClient = CassandraClient.create(
-      vertx,
-      new CassandraClientOptions()
-        .addContactPoint(HOST)
-        .setPort(NATIVE_TRANSPORT_PORT)
-    );
-    Future<Void> future = Future.future();
+    Future<Void> namesKeyspaceInitialized = initializeNamesKeyspace();
+    Future<Void> booksKeyspace = initializeRandomStringKeyspace();
```

Comment:
Where is this one used?

Reply:
when calling `CompositeFuture#all` to wait for keyspaces initialization

Reply:
Right but is it used in other tests?

Reply:
For now, it used in `ExecutionTest# tableHaveSomeRows` and actually it is going to be used in https://github.com/Sammers21/vertx-cassandra-client/pull/14